pkg/frontend/mysql_cmd_executor.go

```diff
@@ -191,6 +198,43 @@ func (o *outputQueue) flush() error {
 	return nil
 }
 
+/*
+handle show create table in plan2 and tae
+*/
+func handleShowCreateTable2(mrs *MysqlResultSet) error {
```

Comment:
I think it is wrong here. As a assumption, the count of the table has 3 fields. Then the `handleShowCreateTable2` will be called 3 times. The  mrs is wrong, I think.